services/app-web/src/components/FileUpload/FileItem/FileItem.tsx

```diff
@@ -120,45 +122,127 @@ export const FileItem = ({
         retryItem(item)
     }
 
-    let statusValue = "";
+    let statusValue = ''
     if (isLoading) {
-        statusValue = "uploading";
+        statusValue = 'uploading'
     } else if (isScanning) {
-        statusValue = "scanning for viruses"
-    } else if (hasDuplicateNameError || hasScanningError || hasUploadError || hasUnexpectedError) {
-        statusValue = "error"
+        statusValue = 'scanning for viruses'
+    } else if (
+        hasDuplicateNameError ||
+        hasScanningError ||
+        hasUploadError ||
+        hasUnexpectedError
+    ) {
+        statusValue = 'error'
     }
 
-    return (
-        <>
-            <div className={styles.fileItemText}>
-                <div
-                    role="progressbar"
-                    aria-valuetext={statusValue}
-                    aria-label={`File status`}
-                >
-                    <img
-                        id={item.id}
-                        data-testid="file-input-preview-image"
-                        src={SPACER_GIF}
-                        alt=""
-                        className={imageClasses}
-                    />
+    if (renderMode === 'list') {
+        return (
+            <>
+                <div className={styles.fileItemText}>
+                    <div
+                        role="progressbar"
+                        aria-valuetext={statusValue}
+                        aria-label={`Status of file ${name}`}
+                    >
+                        <img
+                            id={item.id}
+                            data-testid="file-input-preview-image"
+                            src={SPACER_GIF}
+                            alt=""
+                            className={imageClasses}
+                        />
+                    </div>
+                    <span
+                        style={{
```

Comment:
Hey could you pull this out in the scss module and using classes? This might have been me, its a bad habit I have from rushing during MVP stage. We def don't want inline CSS going on.